website/pages/en/cookbook/timeseries.mdx

```diff
@@ -0,0 +1,179 @@
+---
+title: Subgraph Best Practice 5 - Simplify and Optimize with Timeseries and Aggregations
+---
+
+## TLDR
+
+Leveraging the new timeseries and aggregations feature in subgraphs can significantly enhance both indexing speed and query performance. By offloading aggregation computations to the database and simplifying mapping code, this approach reduces data processing overhead and accelerates queries, especially when dealing with large volumes of time-based data.
+
+## Why Timeseries and Aggregations Are a Best Practice
+
+1. Improved Indexing Time • Less Data to Load: Mappings handle less data since raw data points are stored as immutable timeseries entities. • Database-Managed Aggregations: Aggregations are automatically computed by the database, reducing the workload on the mappings.
+2. Simplified Mapping Code • No Manual Calculations: Developers no longer need to write complex aggregation logic in mappings. • Reduced Complexity: Simplifies code maintenance and minimizes the potential for errors.
+3. Dramatically Faster Queries • Immutable Data: All timeseries data is immutable, enabling efficient storage and retrieval. • Efficient Data Separation: Aggregates are stored separately from raw timeseries data, allowing queries to process significantly less data—often several orders of magnitude less.
+
+## How to Implement Timeseries and Aggregations
+
+### Defining Timeseries Entities
```

Comment:
Is there a hierarchy to all the headings underneath? They're all level 3 headings, so just double checking 😄